Runtime/Scripts/LyumaAv3Runtime.cs

```diff
@@ -3174,8 +2899,12 @@ object GetTypeWithMismatch(AnimatorControllerPlayable playable, int id, Animator
 						if (msgPath.StartsWith("/avatar/parameters/")) {
 							ParamName = msgPath.Split(new char[]{'/'}, 4)[3];
 						}
-						if (BUILTIN_PARAMETERS.Contains( ParamName ) ) {
-							processOSCVRCInputMessage( ParamName, arguments[0]);
+
+						BuiltinParameterDefinition messageParameter =
+							BUILTIN_PARAMETERS.FirstOrDefault(x => x.name == ParamName);
+						if (messageParameter != null) {
```

Comment:
I don't want to cause logspam in this case, since this can happen for various legitimate reasons (for example, relaying output OSC data from VRC into the editor)

So we should make sure this only logs at most once or once per parameter (for example, we could make a bool on the BuiltinParameterDefinition class that keeps track if we already warned.)